### PR TITLE
Tcc flag linx

### PR DESCRIPTION
**Pull request by akshathatc**

### CODE REVIEW

File: Platform/AlderlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c [hunk 1]
```diff
@@ -58,6 +58,7 @@ GetCpuStepping(
   return ((CPU_STEPPING) (Eax.Uint32 & CPUID_FULL_STEPPING));
 }
 
+#if FeaturePcdGet (PcdTccEnabled)
```
Comment (sindhuragrandhi) [inline Platform/AlderlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c:61]: This condition is a duplicate. Same condition already exists when it is being called.

File: Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c [hunk 2]
```diff
@@ -319,6 +319,7 @@ GetMaxCpuPciePortNum (
   }
 }
 
+#if FeaturePcdGet (PcdTccEnabled)
```
Comment (sindhuragrandhi) [inline Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c:322]: This condition is already checked when the function is being called? Why do you have to build out completely?

File: Platform/AlderlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c (hunk 1 above)
Comment (akshathatc) [reply Platform/AlderlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c:61]: It was added to fix build errors caused due to Tcc FSP UPDs which are used in function definition.

File: Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c (hunk 2 above)
Comment (akshathatc) [reply Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c:322]: It was added to fix build errors caused due to Tcc FSP UPDs which are used in function definition.

**Review by sindhuragrandhi — APPROVED** [review]

**Review by gdong1 — APPROVED** [review]

**Pull request merged**